From 37d7bb9cfd2349f41639a4788b095dc4f2d4b0f3 Mon Sep 17 00:00:00 2001
From: Sean Whitton <spwhitton@spwhitton.name>
Date: Wed, 24 Jul 2019 20:05:34 +0100
Subject: [PATCH 1/3] d-m-{merge,debrebase}(7): Use an 'upstream/' tag prefix
 in gbp.conf

This is a useful convention to distinguish actual upstream tags from
upstream tarball-representing tags created by the Debian package
maintainer.

Note that use of this convention is already recommended in section
"Using untagged upstream commits".

Signed-off-by: Sean Whitton <spwhitton@spwhitton.name>
---
 dgit-maint-debrebase.7.pod | 4 +++-
 dgit-maint-merge.7.pod     | 4 +++-
 2 files changed, 6 insertions(+), 2 deletions(-)

diff --git a/dgit-maint-debrebase.7.pod b/dgit-maint-debrebase.7.pod
index 9c9598bb..2ac70b57 100644
--- a/dgit-maint-debrebase.7.pod
+++ b/dgit-maint-debrebase.7.pod
@@ -181,7 +181,7 @@ Now create I<debian/gbp.conf>:
     [DEFAULT]
     upstream-branch = upstream
     debian-branch = master
-    upstream-tag = %(version)s
+    upstream-tag = upstream/%(version)s
 
     sign-tags = True
     pristine-tar = False
@@ -392,6 +392,8 @@ or if you have a working watch file
 
 =back
 
+replacing I<1.2.3> with I<upstream/1.2.3> if you imported a tarball.
+
 This invocation of git-debrebase(1) involves a git rebase.  You may
 need to resolve conflicts if the Debian delta queue does not apply
 cleanly to the new upstream source.
diff --git a/dgit-maint-merge.7.pod b/dgit-maint-merge.7.pod
index 0ccd8c7e..17637bad 100644
--- a/dgit-maint-merge.7.pod
+++ b/dgit-maint-merge.7.pod
@@ -166,7 +166,7 @@ Now create I<debian/gbp.conf>:
     [DEFAULT]
     upstream-branch = upstream
     debian-branch = master
-    upstream-tag = %(version)s
+    upstream-tag = upstream/%(version)s
 
     sign-tags = True
     pristine-tar = False
@@ -393,6 +393,8 @@ or if you have a working watch file
 
 =back
 
+In the following, replace I<1.2.3> with I<upstream/1.2.3>.
+
 =head2 Reviewing & merging the release
 
 It's a good idea to preview the merge of the new upstream release.
-- 
2.20.1

